### PR TITLE
The new landing page with voucher purchase form

### PR DESCRIPTION
**Pull request by OduduU**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -11,9 +11,9 @@ import './helperFunctions/protos';
 import './App.scss';
 import ErrorBoundary from './common/ErrorBoundary';
 import Fallback from './components/Fallback';
-import Home from './pages/Home';
-import Header from './components/Header';
-import Footer from './components/Footer';
+import Home from './pages/Home/newindex';
+// import Header from './components/Header';
```
Comment (Tyav) [inline src/App.js:15]: Remove the commented codes

File: src/App.js [hunk 2]
```diff
@@ -42,7 +42,7 @@ function App() {
     <ErrorBoundary>
       <Provider store={store}>
         <Router>
-          <Header />
+          {/* <Header /> */}
```
Comment (Tyav) [inline src/App.js:45]: Just as above, remove all commented codes

File: src/pages/Home/newindex.scss [hunk 3]
```diff
@@ -0,0 +1,335 @@
...
+                h2 {
+                    margin: 5px 0 23px;
+                    color: #292f35;
+                    font-size: 30px;
+                }
+            }
+
+            .questions-answers {                
+                display: flex;
+                // justify-content: space-between;
```
Comment (Tyav) [inline src/pages/Home/newindex.scss:257]: and here

File: src/App.js (hunk 2 above)
Comment (OduduU) [reply src/App.js:45]: ok

**Pull request merged**